### PR TITLE
Update visual elements

### PR DESCRIPTION
**Pull request by alexbruy**
- make highligh line thicker
- reduce size of vertex markers
- use circles without icon for existing vertices

Before
image

[**UPDATED**] After

### CODE REVIEW

**Comment by tomasMizera** [comment]
@alexbruy can you please update this PR with the latest geo-edit code please?

**Comment by alexbruy** [comment]
Added back button
image
image

File: app/qml/map/MapWrapper.qml [hunk 1]
```diff
@@ -586,57 +587,109 @@ Item {
...
-
-    onClicked: recordingToolsLoader.item.undo()
-
-    maxWidth: InputStyle.mapBtnHeight
-    withImplicitMargins: false
+    maxWidth: parent.width * 0.8
 
-    anchors.bottom: wouldCollideWithLayerBtn() ? activeLayerButton.top : parent.bottom
-    anchors.bottomMargin: root.mapExtentOffset + InputStyle.smallGap
+    anchors.top: howtoEditingBanner.showBanner ? howtoEditingBanner.bottom : parent.top
```
Comment (tomasMizera) [inline app/qml/map/MapWrapper.qml:603]: We need to consider all banners, not just the `howToEditing` here:

**Review by tomasMizera — COMMENTED** [review]
- Have a look at the `back` button during stakeout - it has weird `topMargin` (look at both - long and short mode)
- Handles are much better now, but there is still place for tuning, have a look at the video, these handles are too far when I zoom to ~20m and less

https://user-images.githubusercontent.com/22449698/191178167-393beea9-2e56-409c-8a49-284af2bc58c8.mp4

**Comment by tomasMizera** [comment]
Updated visual, let's merging! Good job @alexbruy

**Pull request merged**